apps/qubit/modules/search/templates/_facet.php

```diff
@@ -1,76 +1,62 @@
-<?php if (isset($sf_request->$facet) || (isset($open) && $open
-  && isset($pager->facets[$facet]) && 0 < count($pager->facets[$facet]['terms']))): ?>
-  <section class="facet open">
-<?php else: ?>
-  <section class="facet">
-<?php endif; ?>
+<?php if ((isset($filters[$facet]) && @count($pager->facets[$facet]['terms']) == 1)
+  || @count($pager->facets[$facet]['terms']) > 1): ?>
```

Comment:
This probably isn't your code but I think we should be more verbose here, using @ instead of checking isset() a bunch seems like really bad programming practice.